src/derived/evaluateOnce.js

```diff
@@ -0,0 +1,33 @@
+import AnimatedValue from '../core/AnimatedValue';
+import { block, call, always, cond, set } from '../base';
+
+/**
+ * evaluate given node and notify children
+ * @param node - node to be evaluated
+ * @param children - children (or one child) nodes to be notified
+ * @param callback - after callback
+ */
+export function evaluateOnce(node, children = [], callback) {
+  if (!Array.isArray(children)) {
+    children = [children];
+  }
+  const done = new AnimatedValue(0);
+  const evalNode = cond(
+    done,
+    0,
+    block([
```

Comment:
don't think you should need to use `block` here. Just pass in the array and it should get converted to block automatically.

Would be even better if instead of using `block` we just have that `call` node directly here and have `node` and `set(done, 1)` as input arguments for the `call`